package.json

```diff
@@ -2944,7 +2944,7 @@
         "@types/node": "16.x",
         "@types/node-fetch": "^2.5.12",
         "@types/semver": "^7.3.9",
-        "@types/vscode": "1.66.0",
+        "@types/vscode": "1.67.0",
```

Comment:
I keep this in sync with the minimum VSCode version because `vsce` can get mad if there's a mismatch.